### PR TITLE
Changes for RISC-V

### PR DESCRIPTION
**Pull request by iarspider**
* libzmq: add explicit dependency on libunwind
* matplotlib: explicitly ask to use system freetype, i.e. not build it from source
* ROOT: update config.{sub,guess} for libAfterImage
* openmpi: explicitly configure to build it's own version of `pmix` (no version is installed on x86/arm machines, the version installed on RISC-V is not accepted because it's built with wrong libevent?)

### CODE REVIEW

**Comment by iarspider** [comment]
please test

**Comment by iarspider** [comment]
please test for el8_aarch64_gcc12

**Comment by iarspider** [comment]
please test for slc7_amd64_gcc12

**Comment by iarspider** [comment]
please test for el9_amd64_gcc12

File: libzmq.spec [hunk 1]
```diff
@@ -3,6 +3,7 @@
 %define tag 5bf04ee2ff207f0eaf34298658fe354ea61e1839
 Source: git+https://github.com/zeromq/libzmq.git?obj=%{branch}/%{tag}&export=%{n}-%{realversion}&output=/%{n}-%{realversion}.tgz
 BuildRequires: autotools
+Requires: libunwind
```
Comment (smuzaffar) [inline libzmq.spec]: @iarspider , I think better to use `--enable-libunwind=no` configure option

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
+externals

**Pull request merged**